akka-actor-testkit-typed/src/main/scala/akka/actor/testkit/typed/javadsl/TestProbe.scala

```diff
@@ -220,6 +220,11 @@ abstract class TestProbe[M] {
    */
   @InternalApi protected def expectMessageClass_internal[C](max: FiniteDuration, c: Class[C]): C
 
+  /**
+   * Receive one message of type `M` within `remainingOrDefault` as the `TestTimeFactor`.
+   */
+  def receiveOne: M
```

Comment:
`()` to remain consistent with the other no param methods

Reply:
I understand the reason but it's a no-arg function ;) Will change for consistency.